cli/cmd/suppressions.go

```diff
@@ -63,4 +68,183 @@ func init() {
 	// gcp
 	suppressionsCommand.AddCommand(suppressionsGcpCmd)
 	suppressionsGcpCmd.AddCommand(suppressionsListGcpCmd)
+	suppressionsGcpCmd.AddCommand(suppressionsMigrateGcpCmd)
+}
+
+func autoConvertSuppressions(convertedPolicyExceptions []map[string]api.PolicyException) {
+	cli.StartProgress("Creating policy exceptions ...")
+	for _, exceptionMap := range convertedPolicyExceptions {
+		for policyId, exception := range exceptionMap {
+			response, err := cli.LwApi.V2.Policy.Exceptions.Create(policyId, exception)
+			if err != nil {
+				cli.Log.Debug(err, "unable to create exception")
+				cli.OutputHuman(color.RedString(
+					"Error creating policy exception to create exception. %e"),
```

Comment:
Not sure this is the correct placeholder for your err.
https://pkg.go.dev/fmt
```suggestion
					"Error creating policy exception to create exception. %s"),
```